pyproject.toml

```diff
@@ -5,7 +5,18 @@ description = "Arduino Library Manager submission parser"
 authors = ["Arduino <info@arduino.cc>"]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.9"
+
+# Integration tests dependencies.
+invoke = "1.4.1"
+pytest = "6.1.2"
 
 [tool.poetry.dev-dependencies]
+black = { version = "^21.4b2", allow-prereleases = true }
 codespell = ">=2.0.0"
+flake8 = "^3.9.1"
+pep8-naming = "^0.11.1"
```

Comment:
I don't think we need the split in dependencies, since we're using this `pyproject.toml` only to run integration tests we could put everything under `depedencies`.

Reply:
I have now made the suggested change here: https://github.com/arduino/library-registry-submission-parser/compare/4dd429572daca5effec51709361a3e5ddb1c1ee2..73ec67ebea1bcfd6b7be13d66e486913c994c8c4